src/Agent.Service/Windows/EventLogger.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Diagnostics;
+
+namespace AgentService
+{
+    public class EventLogger
+    {
+        public const string EventSourceName = "VstsAgentService";
+
+        public static void WriteInfo(string message)
+        {
+            WriteToEventLog(message, EventLogEntryType.Information);
+        }
+
+        public static void WriteException(Exception exception)
+        {
+            WriteToEventLog(exception.ToString(), EventLogEntryType.Error);
+        }
+        
+        public static void WriteToEventLog(string eventText, EventLogEntryType entryType)
+        {
+            EventLog.WriteEntry(EventSourceName, eventText, entryType, 100);
```

Comment:
its been the old pattern. I have just done the refactoring. I will check with Ting on this.

---
In reply to: [115211502](https://github.com/Microsoft/vsts-agent/pull/951#discussion_r115211502) [](ancestors = 115211502)